SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/StaticSubobjectsTest/StaticSubobjectsTest.cpp

```diff
@@ -142,11 +142,7 @@ void AStaticSubobjectsTest::PrepareTest()
 
 	// Step 9
 	AddStep(TEXT("StaticSubobjectsTestClientSeeRightNumberComponentsWithoutWait2"), FWorkerDefinition::Client(1), nullptr, [this]() {
-		AssertIsValid(TestActor->TestStaticComponent1, TEXT("TestStaticComponent1 should be valid."));
```

Comment:
for context to anyone looking at this, this passed on spatial, and failed in native (because while the subobject was not deleted on the client in both spatial and native, the pointer that was nulled on the server as a result was replicated to the client in native and not spatial)